packages/client/package.json

```diff
@@ -72,22 +72,26 @@
   "types": "./dist/src/lib.d.ts",
   "module": "./src/lib.js",
   "main": "./dist/src/lib.cjs",
-  "exports": {
-    ".": {
-      "browser": "./src/lib.js",
-      "require": "./dist/src/lib.cjs",
-      "node": "./src/lib.js"
-    },
-    "./src/platform.js": {
-      "browser": "./src/platform.web.js",
-      "require": "./dist/src/platform.cjs",
-      "node": "./src/platform.js"
-    },
-    "./name": {
-      "browser": "./src/name.js",
-      "require": "./dist/src/name.cjs",
-      "node": "./src/name.js"
-    }
+  "exports": { 
+    ".": { 
+      "browser": "./src/lib.js", 
+      "require": "./dist/src/lib.cjs", 
+      "node": "./src/lib.js" 
+    }, 
+    "./src/platform.js": { 
+      "browser": "./src/platform.web.js", 
+      "require": "./dist/src/platform.cjs", 
+      "node": "./src/platform.js" 
+    }, 
+    "./name": { 
+      "browser": "./src/name.js", 
+      "require": "./dist/src/name.cjs", 
+      "node": "./src/name.js" }, 
+    "./dist/bundle.esm.min.js": { 
+      "browser": "./dist/bundle.esm.min.js", 
+      "require": "./dist/bundle.esm.min.js", 
+      "node": "./dist/bundle.esm.min.js" 
+      }
```

Comment:
🙏 please can you format the closing curlys consistently?

Reply:
Thanks you guys, i was actually hoping to get a microgrant from filecoin for [my project](https://github.com/jilt/varda-vault), but I'm feeling a little lost with the submission process, if there's any filecoin [maintainer](https://github.com/filecoin-project/community/#maintainers), please ping me, I will appreciate some help in writing the grant proposal.
Thanks again!